client/status/container.go

```diff
@@ -25,6 +32,16 @@ func (x ContainerNotFound) Error() string {
 	)
 }
 
+func (x ContainerNotFound) Is(target error) bool {
+	switch target.(type) {
+	default:
+		return false
+	case
+		ContainerNotFound, *ContainerNotFound:
```

Comment:
I'd expect to see this on the same line with `case`. Does `gofmt` change anything here?

Reply:
No, `gofmt` don't mind to such code. But anyway, replaced to one line - it makes sence